third_party/forked/golang/template/funcs.go

```diff
@@ -75,14 +75,6 @@ func addValueFuncs(out map[string]reflect.Value, in FuncMap) {
 	}
 }
 
```

Comment:
prefer leaving the code in third_party/forked alone for things like this

Reply:
Fixing